docs/api.yaml

```diff
@@ -2263,8 +2263,6 @@ paths:
           content:
             application/json:
               schema:
-                type: object
-                description: Standard Response
```

Comment:
More of a question than anything else, but I don't see a top-level `description` in the `Project` schema. Did you remove it here because it just doesn't add much? Have you noticed any trend whereby we should generally be adding `description` in more places or removing it in more places?